packages/eslint-plugin-gestalt/src/helpers/styleValidators.js

```diff
@@ -58,7 +58,8 @@ export function genBointLookup(
   start: number,
   end: number = 12,
 ): GenBointLookupType {
-  const lookupMap = {};
+  const lookupMap: {| [string | number]: string |} = {};
```

Comment:
nit: define this type once and use it in both places

Reply:
DONE